.gitignore

```diff
@@ -3,3 +3,11 @@ _static
 _templates
 
 TODO
+
```

Comment:
Please keep PRs to doing one single thing.

In general here in particular has been "this belongs in your global gitignore, not the one for every repo, which is why it isn't here already", but if you disagree, we should discuss in a separate PR :)

Reply:
My personal opinion is that each repo should ignore the relevant files.
But as you mentioned this goes a bit beyond the real goal of this PR. 
So I'll remove for now the gitignore changes